PsfShimMonitor/KernelTrace.cs

```diff
@@ -0,0 +1,1634 @@
+﻿//-------------------------------------------------------------------------------------------------------
+// Copyright (C) TMurgent Technologies. All rights reserved.path
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+//
+// NOTE: TraceShimMonitor is a "procmon"-like display of events captured via the PSF TraceShim.
+
+using Microsoft.Diagnostics.Tracing;  // consumer
+using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Diagnostics.Tracing.Session; // controller
+using System;
+using System.ComponentModel;  // backgroundworker
+using System.Threading;
+using System.Windows;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;  // ObservableCollection
+
+namespace TraceShimMonitor
+{
+    public partial class MainWindow : Window
+    {
+        public TraceEventSession myTraceEventSession_ProcsKernel = null;
```

Comment:
3 member naming conventions here - can we standardize?

Reply:
Updated 1/25/2019